Introduction to Kotlin Multiplatform/07_js_canvas.md

```diff
@@ -80,8 +85,8 @@ renderToCanvas(document.getElementById("canvas") as HTMLCanvasElement) { image -
 
 It is now time to run the Gradle `run` task to start the server-side Kotlin/JVM project, and the
```

Comment:
It is now time to run the Gradle `run` task to start the server-side Kotlin/JVM project, and the
`jsRun` task to start the client-side Kotlin/JS application. We should be
able to see two images – the one that is rendered server-side with the Kotlin/JVM, and a second
that is rendered directly in the browser with Kotlin/JS:

![](./assets/site-full.png)